docs/zephyr_02_functional_requirements.sdoc

```diff
@@ -448,6 +448,448 @@ Zephyr shall provide an interface to assign a specific handler for a fatal error
 
 [/SECTION]
 
+[SECTION]
+TITLE: System Initialization
+
+[REQUIREMENT]
+STATUS: Draft
+TYPE: Functional
+COMPONENT: System Initialization
+TITLE: Initialization follows dependency order
```

Comment:
"Order of initialization" sounds like a section title rather than an individual requirement title. 

There may be multiple requirements in this category once we allow for components to be initialized in parallel (e.g., NAND file system integrity check in parallel with EEPROM file system integrity check).